lib/instances/LocalizationTable.lua

```diff
@@ -0,0 +1,35 @@
+local BaseInstance = import("./BaseInstance")
+local InstanceProperty = import("../InstanceProperty")
+local json = import("../json")
+
+local CONTENTS = "contents"
+
+local LocalizationTable = BaseInstance:extend("LocalizationTable")
+
+LocalizationTable.properties.SourceLocaleId = InstanceProperty.normal({
+	getDefault = function()
+		return "en-us"
+	end,
+})
+
+function LocalizationTable.prototype:SetContents(contents)
+	getmetatable(self).instance.properties[CONTENTS] = json.decode(contents)
+end
+
+function LocalizationTable.prototype:GetString(targetLocaleId, key)
+	local contents = getmetatable(self).instance.properties[CONTENTS]
```

Comment:
We should put contents straight onto `getmetatable(self).instance` -- it's not well documented, but that's the place to put all of the internal state we need.

Putting this into `properties` collides with the concept of user-visible properties!